.github/workflows/php.yml

```diff
@@ -2,9 +2,9 @@ name: PHP Check
 
 on:
   push:
-   branches:
-      - '*'
-  pull_request:
+    branches:
+      - release-2.1
```

Comment:
Do you know if this means if we branch, will we have to setup a workflow on that branch, or does this one apply to all branches?  Just making sure we understand when we start the next version, how we can modify this workflow to work where its version support will be.

Reply:
I believe we can just add new branches to this list when necessary. I'll let @live627 answer that definitively, though.

Reply:
I can confirm.

The PHP workflow on the new feature branch will be very different: unit tests, static analysiss, etc.